File: api/src/main/java/com/hirrao/javaee/entity/DrugManageResult.java
package com.hirrao.javaee.entity;

import com.baomidou.mybatisplus.annotation.TableField;
import com.baomidou.mybatisplus.annotation.TableId;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.Setter;

import java.sql.Date;
import java.sql.Time;

@Getter
@Setter
@AllArgsConstructor
public class DrugManageResult {
    private Long drugId;
    private String drugName;
    private String frequency;
    private String unit;
    private Float dosage;
    private int isActive;

    private Long alertId;
    private Time alertTime;
    private Date eatTime;
}

File: api/src/main/kotlin/cn/hirrao/javaee/entity/DrugManageResult.kt
package cn.hirrao.javaee.entity

import java.time.LocalDate
import java.time.LocalTime

data class DrugManageResult(
    val drugId: Long? = null,
    val drugName: String? = null,
    val frequency: String? = null,
    val unit: String? = null,
    val dosage: Float? = null,
    val isActive: Int = 0,
    val alertId: Long? = null,
    val alertTime: LocalTime? = null,
    val eatTime: LocalDate? = null,
)
